packages/flutter/lib/src/widgets/scroll_position.dart

```diff
@@ -307,7 +308,7 @@ class ScrollPosition extends ViewportOffset {
 
   @override
   void correctBy(double correction) {
-    _pixels += correction;
+    correctPixels(_pixels + correction);
```

Comment:
The idea was that one could just override correctPixels and see all the corrections.